### DISCUSSION TITLE
Lodestar Standup 2024-4 (January 30, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6349] [Standups]
### Meeting Info

[Jan 30, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/jan-30-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - 1.15.1 hotfix release
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by wemeetagain** [comment]
Lets list and discuss areas where we've specialized individually, and how we can spread that knowledge thru the team.

**Comment by wemeetagain** [comment]
Lets discuss our strategy for management of open PRs.
Should we have draft PRs open that aren't close to review status -- that aren't ready for feedback?
Can we use issues to promote visibility of branches that don't need feedback?

**Comment by twoeths** [comment]
Just want to make sure we don't miss this, Nethermind has a metric to track `notifyNewPayload` time to reach execution engine, we should improve it. That should help improve attestation performance as well.
cc @g11tech @philknows

**Comment by twoeths** [comment]
also how to speed up EIP-6110 work of @ensi321

**Reply by ensi321** [reply]
Thanks @tuyennhv . Was about to say this.

Mainly want to discuss the EIP-6110 PR from several perspectives:
1) How to make it easier for review (Discussion session? Split it into smaller chunks?) especially when I will be away for two weeks and unavailable to answer questions in a timely manner. The original intent of this PR is to include everything controversial so everyone can discuss. There is a follow up PR planned for other things like renaming, refactoring etc.
2) How to manage it along with other Electra EIPs? Now that 6110 is scheduled for Electra, and @g11tech just created a new branch `electra-fork` #6352 . Should it first merge into the electra branch before `unstable`?

A further discussion that's nice to have is to establish the process of develop, manage and test the Electra EIPs. 
eg. PR merge to `electra-fork` first then test or test at the time of the PR?

**Reply by g11tech** [reply]
yes, I will actively maintain and manage electra-fork and its participation into the devnets so you can consider me sort of the "fork-owner" as this kind of structure has served us well in past forks.

all the work for the fork will need to be build against and merged into the branch till we are running devnets on it.

then we can extract the work into the unstable.

**Reply by g11tech** [reply]
so as a first step, i will review and rebase 6110 work and get it merged ready into this

**Reply by g11tech** [reply]
(and please never do `merge` in your branch, always `rebase` as it really muddies the commit history and makes everything difficult)

**Comment by matthewkeil** [comment]
I added a discussion for how we handle skipped tests.

https://github.com/ChainSafe/lodestar/discussions/6376

**Comment by nflaig** [comment]
If there is time left could quickly talk about what we should do with the env vars issue on k8s
- https://github.com/ChainSafe/lodestar/issues/6045

**Reply by nflaig** [reply]
We discussed to add this to a known issues page in our docs
- https://github.com/ChainSafe/lodestar/issues/6378

**Comment by jeluard** [comment]
I would be interested in discussing the high-level strategy for 2024.

**Reply by philknows** [reply]
Very high level, but can give a brief summary on what was discussed during our last team retreat:

Priority 1: Adoption and Legitimacy
Priority 2: Research and Development
Priority 3: Improve Developer Experience
Priority 4: Performance Improvements
Priority 5: Improve User Experience

**Comment by philknows** [comment]
Closed in favour of #6390
